### PR TITLE
Update doc-guidelines.md

### PR DESCRIPTION
**Pull request by BirdboyBolu**
I made some changes to the sentences to improve readability

### CODE REVIEW

File: docs/doc-guidelines.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-If you're new to writing or editing ownCloud documentation, we recommend the [Getting Started Guide](https://github.com/owncloud/docs/blob/master/docs/getting-started.md) and [Building the Documentation](https://github.com/owncloud/docs/blob/master/docs/build-the-docs.md) so you get a good understanding what awaits you. (Just don't let it discourage you!). Keep the [Best Practices Guide](https://github.com/owncloud/docs/blob/master/docs/best-practices.md) collection in mind.
+If you're new to writing or editing ownCloud documentation, we recommend the [Getting Started Guide](https://github.com/owncloud/docs/blob/master/docs/getting-started.md) and [Building the Documentation](https://github.com/owncloud/docs/blob/master/docs/build-the-docs.md) so you get a good understanding of what awaits you. (Just don't let it discourage you!). Keep the [Best Practices Guide](https://github.com/owncloud/docs/blob/master/docs/best-practices.md) collection in mind.
```
Comment (mmattel) [inline docs/doc-guidelines.md:9]:
```suggestion
If you're new to writing or editing ownCloud documentation, we recommend the [Getting Started Guide](https://github.com/owncloud/docs/blob/master/docs/getting-started.md) and [Building the Documentation](https://github.com/owncloud/docs/blob/master/docs/build-the-docs.md) to get a good understanding of what awaits you. (Just don't let it discourage you!). Keep the [Best Practices Guide](https://github.com/owncloud/docs/blob/master/docs/best-practices.md) collection in mind.
```

**Comment by mmattel** [comment]
Thanks a lot for taking care 👍

**Review by mmattel — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Comment by mmattel** [comment]
> This made me look through the website again and I noticed that OwnCloud has an opening for a Technical Writer role. I would love to fill the position by joining OwnCloud and apply my skills and dedication to ensure that OwnCloud's documentation in top notch.

Just click the apply button and send your application 😄

**Comment by BirdboyBolu** [comment]
Lol, I did that already in May.  I didn't get a response. I just sent another application.

**Pull request merged**